languages/csharp/config.json

```diff
@@ -157,8 +157,8 @@
       {
         "slug": "integral-numbers",
         "uuid": "86bbc073-1726-423d-b156-c394554a4a8c",
-        "concepts": ["casting", "integral-numbers"],
-        "prerequisites": ["arrays", "numbers"]
+        "concepts": ["integral-numbers"],
+        "prerequisites": ["arrays", "casting", "numbers"]
```

Comment:
While conceptually this makes sense, the `casting` concept is unlocked by the `casting` exercise, which has dependencies on `"exceptions", "inheritance", "numbers"` which is a bit heavy. I've looked at the `integral-numbers` exercise and it just requires a basic cast (no `is` or `as` involved). The `numbers` exercise _also_ requires this, so just having that as a prerequisite and not having `casting` could work. What do you think?

I did notice that the `numbers` exercise did not show the syntax for doing the cast, which is a mistake on our part.

Reply:
removed `casting` as a prerequisite and added issue #2262 to improve `numbers` exercise.